Sources/ClientRuntime/ClockSkew/DefaultClockSkewProvider.swift

```diff
@@ -0,0 +1,25 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import struct Foundation.Date
+import struct Foundation.TimeInterval
+import class SmithyHTTPAPI.HTTPRequest
+import class SmithyHTTPAPI.HTTPResponse
+
+public enum DefaultClockSkewProvider {
+
```

Comment:
This is the "generic Smithy" clock skew provider.  It does not do anything.

aws-sdk-swift provides its own implementation of this which actually does measure clock skew.